tests/test_integrated_channels/test_api/test_canvas/test_views.py

```diff
@@ -33,8 +33,8 @@ def setUp(self):
 
         self.enterprise_customer_conf = factories.CanvasEnterpriseCustomerConfigurationFactory(
             enterprise_customer=self.enterprise_customer,
-            client_id='ayy',
-            client_secret='lmao',
+            encrypted_client_id='ayy',
+            encrypted_client_secret='lmao',
```

Comment:
shouldn't we use "decrypted_client_id" and "decrypted_client_secret" to populate a test config?